src/ansys/dpf/core/mesh_info.py

```diff
@@ -0,0 +1,193 @@
+
+        Returns
+        -------
+        :class:`ansys.dpf.core.generic_data_container.GenericDataContainer`
+
+        """
+
+        return self._generic_data_container
+
+    @generic_data_container.setter
+    def generic_data_container(self, value: GenericDataContainer):
+        """GenericDataContainer wrapped into the MeshInfo
+        that contains all the relative information of the derived class.
+
+        Returns
+        -------
+        :class:`ansys.dpf.core.generic_data_container.GenericDataContainer`
+
+        """
+
+        if type(value) is not GenericDataContainer:
+            raise ValueError("Input value must be a GenericDataContainer.")
+        self._generic_data_container = value
+
+    def deep_copy(self, server=None):
+        """Create a deep copy of the scoping's data on a given server.
+
+        This method is useful for passing data from one server instance to another.
+
+        Parameters
+        ----------
+        server : ansys.dpf.core.server, optional
+            Server with the channel connected to the remote or local instance.
+            The default is ``None``, in which case an attempt is made to use the
+            global server.
+
+        Returns
+        -------
+        scoping_copy : Scoping
+        """
+        mesh_info = MeshInfo(server=server)
+        mesh_info._generic_data_container = self.generic_data_container
+        return mesh_info
+
+    def get_property(self, property_name, output_type):
+        """Get property with given name.
+
+        Parameters
+        ----------
+        property_name : str
+            Property name.
+        output_type :  :class:`ansys.dpf.core.common.types`
+
+        Returns
+        -------
+        type
+            Property object instance.
+        """
+        return self.generic_data_container.get_property(property_name, output_type)
+
+    def set_property(self, property_name, prop):
+        """Register given property with the given name.
+
+        Parameters
+        ----------
+        property_name : str
+            Property name.
+        prop : Int, String, Float, Field, StringField, GenericDataContainer, Scoping
+            object instance.
+        """
+
+        return self.generic_data_container.set_property(property_name, prop)
+
+    def get_number_nodes(self):
+        """
+        Returns
+        -------
+        number_nodes : int
+            Number of nodes of the mesh.
+        """
+
+        return self.generic_data_container.get_property("num_nodes", int)
+
+    def get_number_elements(self):
+        """
+        Returns
+        -------
+        number_elements : int
+            Number of elements of the mesh.
+        """
+
+        return self.generic_data_container.get_property("num_elements", int)
+
+    def get_splittable_by(self):
+        """
+        Returns
+        -------
+        splittable by which entity : StringField
+                Name of mesh subdivisions.
```

Comment:
This is not exactly that. I would say: `Name of the properties according to which the mesh can be split by`